pandas/tests/io/test_encoding_aliases.py

```diff
@@ -0,0 +1,32 @@
+import pandas, os, nose
+
+def test_read_csv():
+    # see gh issue 13549
+    test_encodings = ['utf-16', 'utf_16', 'UTF_16', 'UTF-16']
+    engines = ['c', 'python', None]
+    expected_output = [1]*len(test_encodings)*len(engines)
+    test_output = []
+    path = "test.csv"
+    pandas.DataFrame({"A": [0,1], "B": [2,3]}).to_csv(
+        path, encoding="utf-16")
+
+    for encoding in test_encodings:
+        for engine in engines:
+            try:
+                pandas.io.parsers.read_csv(
+                    path,
+                    engine='c',
+                    encoding=encoding)
+                print(encoding, 'succeeded with engine =', engine)
```

Comment:
prepare the expected dataframe and compare the result using ``tm.assert_frame_equal``.